src/library_pthread.js

```diff
@@ -169,9 +169,7 @@ var LibraryPThread = {
 #endif
 
 #if !MINIMAL_RUNTIME
-    setExitStatus: (status) => {
-      EXITSTATUS = status;
-    },
+    setExitStatus: (status) => EXITSTATUS = status,
```

Comment:
It probably doesn't matter for how these particular functions are used, but note that such change changes the return type / value as now `setExitStatus` will return the exit status and `registerTLSInit` will return the new length of array.

Just wanted to confirm it's intended.

Reply:
I do know about that yes, and my assumption is that going from returning nothing to returning something in these cases is likely harmless.  

Perhaps there will be cases where it does matter but I don't see it mattering in these cases.